docs/reference/tags/experience_date_search_tag/index.md

```diff
@@ -0,0 +1,171 @@
+<p>Beyond Asia</p>
+<div>
+    112 Results
+    <span class="prev">
+    <a href="http://beyondadventures.com/pagination-test?page=2">Previous</a>
+    </span>
+    <span class="page">
+    <a href="http://beyondadventures.com/pagination-test?page=1">1</a>
+    </span>
+    <span class="page">
+    <a href="http://beyondadventures.com/pagination-test?page=2">2</a>
+    </span>
+    <span class="page current">3</span>
+    <span class="page">
+    <a href="http://beyondadventures.com/pagination-test?page=4">4</a>
+    </span>
+    <span class="page">
+    <a href="http://beyondadventures.com/pagination-test?page=5">5</a>
+    </span>
+    <span class="deco">&hellip;</span>
+    <span class="page">
+    <a href="http://beyondadventures.com/pagination-test?page=9">9</a>
+    </span>
+    <span class="next">
+    <a href="http://beyondadventures.com/pagination-test?page=4">Next</a>
+    </span>
+</div>
+```
+{% endraw %}
+
+The tag returns an `items` array and a `Paginate` pagination object to the block passed in, the items array represents 1 page of the results from the search.
+
+
+## Pagination
+
+The results of this search are paginated for performance reasons.
+You can paginate results into equal page sizes, defining the number of results to display per page using the `page_size` attribute. There is a maximum page size of 100.
+Alternatively, you can paginate results by month, by setting the `page_size` to `month`.
+
+## Search Params
+
+Search params can be handled in one of two ways:
+
+### Passing explicit attributes
+
+It's possible to use any of the following attributes in the search tag itself, this is useful for building a page responsible for a specific type of product.
+
+#### active_promotion
+Passing `true` to this will mean all items returned will have a currently active promotion.
+
+#### category
+Will search for products which match the category passed, this must use one of Easol's predefined categories and will accept either a single Category or an array.
+
+#### country
+This attribute accepts either Country Codes or Country Names and can be passed as a single attribute or as an array, e.g. `country: ['FR', 'DE']`
+
+#### departure_date
+This attribute takes an object which specifies how to handle the search, through `equal_to` `greater_than` `greater_or_equal_than` or `less_than` each taking a date in SQL format `YYYY-MM-DD` e.g. `departure_date: {greater_than: 'now', less_than: '2019-12-28' }`
+
+#### departure_month
+This allows you to specify which month the trips should depart in, e.g. passing `departure_date: 'Apr'` would return any trips departing in april 2020, april 2021 ..., You can also pass the number of the month i.e. `departure_month: 4` would return the same results.
+
+#### duration
+This attribute takes an object which specifies how to handle the search, through `equal_to` `greater_than` or `less_than` each taking a number of days e.g. `duration: {greater_than: 3, less_than: 8 }`
```

Comment:
The value passed will default to a duration unit of "day".
To specify another duration we need to pass the `duration_unit` i.e. "minute", or "hour".